framework/filters/HostControl.php

```diff
@@ -132,6 +139,11 @@ public function beforeAction($action)
             }
         }
 
+        // replace invalid host info to prevent using it in further processing
```

Comment:
it's fine as is. 

@yiisoft/core-developers Also I just noticed that for such minor things one can edit the file directly on github and instead of commenting, you can just fix it :)